app/models/result_link.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ResultLink < ApplicationRecord
+  belongs_to :keyword, inverse_of: :result_links
+
+  enum link_type: { ads_top: 0, ads_page: 1, non_ads: 2 }
+
+  validates :url, presence: true, allow_blank: false
+  validates :keyword, presence: true
```

Comment:
Do we need this? Association are already validated for presence in `belongs_to`?

Reply:
![image](https://user-images.githubusercontent.com/77609814/124439894-2b8d7d00-dda4-11eb-85b7-979b0985f7e4.png)
Rails is telling me we don't! :P 
Well catched!!!